templates/candidates-single.html

```diff
@@ -58,8 +58,8 @@ <h4 class=>Party</h4>
             {% if leadership_committees %}
             <dl>
               <dt>Leadership PACs</dt>
-              {% for committee in leadership_committees %}
-              <dd><a {% if committee.name_url %}href="{{ committee.name_url }}"{% endif %}>{{ committee.name }}</a></dd>
+              {% for c_id in leadership_committees %}
+              <dd><a {% if leadership_committees[c_id].url %}href="{{ leadership_committees[c_id].url }}"{% endif %}>{{ leadership_committees[c_id].name }}</a></dd>
```

Comment:
Sorry to miss this -- how should this appear if the committee has no url? Perhaps, the the `a` be a `span`. Perhaps there shouldn't be a `dd` at all. Perhaps there's enough confidence that the url will always be present, in which case, there's no need for the "if" at all. I'm not sure; just noting that this implies the corner case is addressed when it's hasn't been.